ffi_c/calling_c.md

```diff
@@ -1,5 +1,3 @@
-**extern.vale**: demonstrates the use of the extern keyword - sqrt() is an external function defined in C (libc)
```

Comment:
This version adds prefixes to any extern, which means we mostly can't call directly into C externs anymore. We could reintroduce this ability one day, if demand is high. Until then, we'll have to write wrapper functions for any FFI.